src/app/core/components/left-menu/left-menu.component.html

```diff
@@ -44,53 +51,61 @@
       [routerLink]="['/myfiles']"
       [class.active]="checkMenuItemActive('/myfiles')"
       prDragTargetRouterLink>
-        <i class="ion-md-folder"></i>
-        My Files
+        <img class="svg-icon" src="assets/svg/left-menu/private-files-icon.svg" />
+        Private Files
       </a>
       <a class="menu-item" [routerLink]="['/shares']" [class.active]="checkMenuItemActive('/shares')">
-        <i class="ion-md-share"></i>
-        Shares
+        <img class="svg-icon" src="assets/svg/left-menu/shared-files-icon.svg" />
+        Shared Files
       </a>
       <a class="menu-item"
         [routerLink]="['/public']"
         [class.active]="checkMenuItemActive('/public')"
         prDragTargetRouterLink>
-        <i class="ion-md-globe"></i>
-        Public
+        <img class="svg-icon" src="assets/svg/left-menu/public-files-icon.svg" />
+        Public Files
       </a>
-      <a class="menu-item" [routerLink]="['/apps']" [class.active]="checkMenuItemActive('/apps')">
-        <i class="ion-md-swap"></i>
-        Apps
+      <hr class="hide-mobile" />
+      <div class="menu-item apps-menu hide-mobile" [class.active]="checkMenuItemActive('/apps')">
+        <a class="apps-link-main" [routerLink]="['/apps']">
+          <img class="svg-icon" src="assets/svg/left-menu/app-files-icon.svg" />
+          <span>App Files</span>
+        </a>
+        <div class="apps-toggle" [class.reverse]="showAppsSubfolders" (click)="showAppsSubfolders = !showAppsSubfolders">
+          <i class="material-icons">expand_more</i>
```

Comment:
Should we show the expand icon if there are no app folders? Feels a little weird to click and just have the icon spin around.

Reply:
Nice, thank you!